chromium/browser_type.go

```diff
@@ -312,13 +312,21 @@ func prepareFlags(lopts *common.LaunchOptions, k6opts *k6lib.Options) map[string
 		f["mute-audio"] = true
 		f["blink-settings"] = "primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4"
 	}
+	ignoreDefaultFlags(f, lopts.IgnoreDefaultArgs)
 
 	setFlagsFromArgs(f, lopts.Args)
 	setFlagsFromK6Options(f, k6opts)
 
 	return f
 }
 
+// ignoreDefaultFlags ignores any flags in teh provide slice
+func ignoreDefaultFlags(flags map[string]interface{}, toIgnore []string) {
+	for _, name := range toIgnore {
+		delete(flags, strings.TrimLeft(name, "--"))
```

Comment:
`TrimLeft` is for removing leading characters (cutset), not for removing a prefix string. So, this would work too:

```
delete(flags, strings.TrimLeft(name, "-"))
```

However, I believe it's better to use `strings.TrimPrefix` for removing a string prefix. From the `strings.TrimLeft` documentation:

```go
// ...
// To remove a prefix, use TrimPrefix instead.
func TrimLeft(s, cutset string) string
```

```suggestion
		delete(flags, strings.TrimPrefix(name, "--"))
```